### ISSUE TITLE
TagHelper contextual C# attribute coloring

### ISSUE THREAD

**Issue by NTaylorMullen** [dotnet/aspnetcore#14290]
TagHelpers can turn HTML attributes into C# aware attributes. This means we'll need to color those attributes accordingly.

The Html/element coloring of TagHelpers is handled by https://github.com/aspnet/Razor.VSCode/issues/20.

**Comment by yszhong12345** [comment]
good

**Comment by NTaylorMullen** [comment]
VSCode has some proposed APIs for this (not part of the LSP spec yet): https://github.com/microsoft/vscode-extension-samples/tree/master/semantic-tokens-sample

**Comment by NTaylorMullen** [comment]
The ability to complete this issue will be blocked on O#'s support of semantic colorization at the LSP layer. That work may spin off of https://github.com/OmniSharp/omnisharp-vscode/issues/3565

**Comment by NTaylorMullen** [comment]
Will be handled by C# semantic colorization
